torch/testing/_internal/common_methods_invocations.py

```diff
@@ -220,6 +221,33 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                                         requires_grad=requires_grad)),)
 
 
+def fast_growing_function_wrapper(fn):
```

Comment:
np_integer_promotion_wrapper

Since this wrapper is only for NumPy references, and is applicable when the NumPy function promotes integers to float. It isn't actually intended to wrap all "fast growing" reference functions.

Reply:
Hmm right. Updated.